tasks/component-compare.js

```diff
@@ -438,10 +458,10 @@ async function main(
 		});
 
 		// This is our report header to indicate the start of a new component's data
-		log.write(`\n${_.pad(` ${component} `, 20, "-").cyan}\n`);
+		cliOutput.push(`\n${_.pad(` ${component} `, 20, "-").cyan}\n`);
 
 		if (warnings.length > 0) {
-			warnings.forEach((warning) => log.write(`${warning}\n`));
+			warnings.forEach((warning) => cliOutput.push(`${warning}\n`));
```

Comment:
I take it some of these changes are just old fashioned refactors? I don't think I should expect to see anything different in my dev console, correct?

Reply:
No output difference, yes. This method of pushing the output to an array and printing it at the end means we can run in parallel in the script without mixing up the resulting data. Before it was writing immediately to the log so we either had to run them one-by-one or risk mixing up the outputs.